CogNative/testUI/UI.py

```diff
@@ -63,31 +60,108 @@ def displayProgressBar():
         pb1["value"] += 20
         sleep(0.25)
     pb1.destroy()
-    Label(ws, text="File Prepared Successfully!", foreground="green").grid(
-        row=8, columnspan=3, pady=10
-    )
 
 
-def cloneVoice(src_lang_var, text_lang_var):
-    data["input"]["src_lang"] = src_lang_var.get()
+def cloneVoice():
     data["input"]["use_embedding"] = embed_check.get()
-
-    data["text"]["text_lang"] = text_lang_var.get()
     data["text"]["text"] = text_synthesize_entry.get()
 
-    # If the path has been set already because the directory was selected
-    if data["output"]["output_audio_path"]:
-        data["output"]["output_audio_path"] = (
-            data["output"]["output_audio_path"] + "/" + output_name_entry.get()
-        )
-    # If it hasn't set directory to file name
+    output_dir = data["output"]["output_dir"]
+    data["output"]["output_audio_path"] = f"{output_dir}/{output_name_entry.get()}"
+    output_dir_lbl.config(text=data["output"]["output_audio_path"])
+
+    cmd = ['python', '-m', 'CogNative.main',
+           '-sampleAudio', data["input"]["audio_to_clone"],
+           '-out', data["output"]["output_audio_path"],
+           '-useExistingEmbed', 'y'

```

Comment:
```suggestion
           '-useExistingEmbed', data["input"]["use_embedding"]
```

This should fix the issue.